src/xenia/app/xenia_main.cc

```diff
@@ -91,6 +91,8 @@ DEFINE_bool(mount_scratch, false, "Enable scratch mount", "Storage");
 DEFINE_bool(mount_cache, true, "Enable cache mount", "Storage");
 UPDATE_from_bool(mount_cache, 2024, 8, 31, 20, false);
 
+DEFINE_bool(mount_devkit, false, "Enable devkit mount", "Storage");
```

Comment:
Maybe in this case it should be called force_mount_devkit because normally game must request mounting of devkit partition